docs/guide/apm-data-model.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[apm-data-model]]
+== Data Model
+
+Elastic APM agents capture different types of information from within their instrumented applications - namely `transactions`, `spans`, and `errors`. 
+
+* <<transactions,Transaction>>
+* <<transaction-spans,Spans>>
+* <<errors,Errors>>
+
+[[transactions]]
+=== Transactions
+
+A transaction describes an event captured by an Elastic APM agent instrumenting a service. Some examples of a transaction might be:
+
+* An HTTP request
+* A background job
+
+A transaction contains:
+
+* The timestamp and duration of the event
+* A unique id, type, and name
+* A result (e.g. a response code)
+* Some contextual data (see below for details)
+* Other relevant information depending on the agent. Example: The JavaScript RUM captures transaction marks,
+which are points in time relative to the start of the transaction with some label.
+
+[[transactions-context]]
+include::../context.asciidoc[]
+
+Agents decide whether to sample transactions or not,
+and provide settings to control sampling behavior.
+If sampled,
+the {apm-server-ref}/transaction-spans.html[spans] of a transaction are sent and stored as separate documents. Within one transaction there can be several, or no spans captured. 
+
+Transactions are stored in {apm-server-ref}/transaction-indices.html[transaction indices].
+
+[[transaction-spans]]
+=== Spans
+Transactions can have 0, 1, or many spans. Spans have a `transaction.id` attribute that refer to their parent transaction.
```

Comment:
they also have a parent id attribute that can be another span or that transaction